### ISSUE TITLE
Batcache rules applying to VIP Go ruleset?

### ISSUE THREAD

**Issue by mikeyarce** [Automattic/VIP-Coding-Standards#266]
## Bug Description

I'm seeing PHPCS Warnings like this on the WordPress-VIP-Go ruleset:

> Batcache whitelisted GET param, `utm_source`, found. Batcache whitelisted parameters get stripped and are not available in PHP.

## Minimal Code Snippet
 
Example code: 

```php
 $utm_source   = isset($_GET[  'utm_source']) ? sanitize_text_field( $_GET['utm_source'] ) : '';
```

I think this is because WordPress-VIP-Go includes WordPressVIPMinimum, right?   Should we do something to not include the `batcache` specific rules? 

- [x] I have verified the issue still exists in the `master` branch of VIPCS.

**Comment by GaryJones** [comment]
> this is because WordPress-VIP-Go includes WordPressVIPMinimum, right?

[Correct](https://github.com/Automattic/VIP-Coding-Standards/blob/a1dc1b749aff6679fa65dc717434697ce35dd8c9/WordPress-VIP-Go/ruleset.xml#L6).



> Should we do something to not include the `batcache` specific rules?

We already have a [section for disabling `WordPressVIPMinimum`](https://github.com/Automattic/VIP-Coding-Standards/blob/a1dc1b749aff6679fa65dc717434697ce35dd8c9/WordPress-VIP-Go/ruleset.xml#L320-L340) rules, so the [Batcache Sniff](https://github.com/Automattic/VIP-Coding-Standards/blob/master/WordPressVIPMinimum/Sniffs/Cache/BatcacheWhitelistedParamsSniff.php) could be added as a `severity` `0` there, along with a comment explaining that Batcache isn't used on the VIP-Go platform.
